src/Compilers/CSharp/Test/Semantic/FlowAnalysis/RegionAnalysisTests.cs

```diff
@@ -165,6 +165,43 @@ static void Main(string[] args)
             Assert.Equal("args, x, y", GetSymbolNamesJoined(analysis.DefinitelyAssignedOnExit));
         }
 
+        [Fact]
+        public void TestDataFlowsOfIdentifierWithDelegateConversion()
+        {
+            var analysis = CompileAndAnalyzeDataFlowExpression(@"
+using System;
+
+namespace ICSharpCode
+{
+	internal static class NoExtensionMethods
+	{
+		internal static Func<T> AsFunc<T>(this T value) where T : class
+		{
+			return new Func<T>(/*<bind>*/ value /*</bind>*/.Return);
```

Comment:
Let's also test a method group conversion scenario, i.e. instead of `new Func<T>`, we cast to `Func<T>`.